src/rooms/rooms.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { Body, Controller, Get, Param, Post } from '@nestjs/common';
+import { RoomsService } from './rooms.service.js';
+
+@Controller()
+export class RoomsController {
+  constructor(private readonly rooms: RoomsService) {}
+
+  @Post('rooms')
+  create(@Body() body: { targetUrl: string }) {
+    const roomRecord = this.rooms.createRoom(body.targetUrl);
+    return {
+      roomId: roomRecord.roomId,
+      shareToken: roomRecord.token,
+      shareUrl: `http://localhost:3000/s/${roomRecord.token}`,
```

Comment:
P1;
추후 리팩토링을 고려하여 ConfigService를 사용하는 것은 어떨까요?

```typescript
import { ConfigService } from '@nestjs/config';

@Post('rooms')
create(@Body() body: { targetUrl: string }) {

  const baseUrl = this.config.get<string>('BASE_URL') ?? 'http://localhost:3000';

  return {
    shareUrl: `${baseUrl}/share/${roomRecord.token}`, 
  };
}
```

```suggestion
      shareUrl: `${baseUrl}/s/${roomRecord.token}`,
```

Reply:
좋은 제안 감사합니다. 배포 시 base URL이 달라질 수 있어서 ConfigService(예: WEB_BASE_URL)로 분리하는 방향으로 반영하겠습니다. MVP 단계에서는 기본값을 localhost로 두고요.